.golangci.yaml

```diff
@@ -13,7 +13,9 @@ linters:
     - ineffassign
     - golint
     - goconst
-    - goimports #- unused, https://github.com/golangci/golangci-lint/issues/337#issuecomment-536721714
+    - gofmt
+    - goimports
+    - unused
```

Comment:
Has this issue been resolved in 1.30.0?

Reply:
I vaguely remember this was an issue with Travis, it was running out of memory there.

Reply:
ah ok yea it was removed in https://github.com/open-policy-agent/gatekeeper/pull/511